bullet_train/docs/heroku.md

```diff
@@ -4,24 +4,29 @@ When you're ready to deploy to Heroku, it's highly recommended you use this butt
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=http://github.com/bullet-train-co/bullet_train)
 
-This button leverages the configuration found in `app.json`, including sensible production-ready defaults for dyno formation, third-party services, buildpack configuration, etc. The resources provisioned will cost about **$140/month**.
+This button leverages the configuration found in `app.json`, including sensible demo-ready defaults for dyno formation, third-party services, buildpack configuration, etc.
+
+The resources provisioned will cost about **$22/month**.
+
+**Please note:** The resources provisionede via `app.json` are intended to be used for quickly launching your brand new app so that you can demo it easily. When you're ready to go into production you'll want to make some changes to these resources. See the section at the bottom of this page.
```

Comment:
```suggestion
**Please note:** The resources provisioned via `app.json` are intended to be used for quickly launching your brand new app so that you can demo it easily. When you're ready to go into production you'll want to make some changes to these resources. See the section at the bottom of this page.
```